index.html

```diff
@@ -1,12 +1,476 @@
+                                    galley of type."</p>
+                            </div>
+                            <div class="column-img">
+                                <img src="./img/user.png" />
+    
+                            </div>
+                            <div class="column-full-name">
+                                <h3 class="name">William Todd</h3>
+                            </div>
+                            <div class="column-job">
+                                <p>General Counsel and CCO</p>
+                            </div>
+                        </div>
+                        <div class="flex-column-align-center slide">
+                            <div class="column-desc">
+                                <p>
+                                    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                                been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                                galley of type."
+                                "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                                been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                                galley of type."
+                                </p>
+                            </div>
+                            <div class="column-img">
+                                <img src="./img/user.png" />
+    
+                            </div>
+                            <div class="column-full-name">
+                                <h3 class="name">Mark Todd</h3>
+                            </div>
+                            <div class="column-job">
+                                <p>General Counsel and CCO</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                
+            </div>
+            
+            
+        </section>
+
+        <section class="contact-us section-margin">
+            <div class="container">
+                <div class="content">
+                    <div class="flex">
+
+                        <div class="column">
+                            <div class="column-img">
+                                <img src="img/illustration-contact-us.svg" />
+                            </div>          
+                        </div>
+
+                        <div class = "column flex-column">
+                            <div class="column-title">
+                                <h2>Contact Us</h2>
+                            </div>
+                            <div class="column-desc">
+                                <p>We believe in the power of conversations. Start one with us today!</p>
+                            </div>
+
+                            <div class="form">
```

Comment:
pls update structure here. Currently, it's incorrect in terms of html and accessibility. Let's think more about scalability. Let me know if you need help in updating these forms